src/runtime/components/CldImage.vue

```diff
@@ -3,187 +3,189 @@
 // Come back to this after https://github.com/nuxt/nuxt/issues/20936 is fixed
 // import type { AssetOptions } from "@cloudinary-util/url-loader";
 // import type { ConfigOptions } from "@cloudinary-util/url-loader";
-import { useCldImageUrl } from "../composables/useCldImageUrl";
-import { ref } from "vue";
-import { Image } from "@unpic/vue";
+import { ref } from 'vue'
+import { Image } from '@unpic/vue'
+import type { ConstructUrlProps } from '@cloudinary-util/url-loader'
+import { useCldImageUrl } from '../composables/useCldImageUrl'
 
 interface AssetOptionsResize {
-  crop?: string;
-  width?: number | string;
+  crop?: string
```

Comment:
im not sure about the rest of the types but this is now wrong. this was the biggest changes of the URL Loader update. this now additionally accepts an object